website/content/en/v1.2/concepts/disruption.md

```diff
@@ -358,6 +358,22 @@ spec:
         karpenter.sh/do-not-disrupt: "true"
 ```
 
+Pods with PDBs will not be gracefully evicted by the [Termination Controller]({{<ref "#termination-controller">}}) or be considerd for consolidation. When multiple pods on a node have different PDBs, all PDB constraints must be satisfied simultaneously for Karpenter to consolidate the node. This can create complex eviction scenarios:
```

Comment:
I think this could mislead users into thinking all pods on the node must be drainable at a point of time to be considered for consolidation. Really, we just need to be able to drain one pod per PDB.
```suggestion
Pods with PDBs will not be gracefully evicted by the [Termination Controller]({{<ref "#termination-controller">}}) or be considerd for consolidation. When multiple pods on a node have different PDBs, none of the PDBs may be blocking for Karpenter to consolidate the node. This can create complex eviction scenarios:
```